tools/linker/MonoTouch.Tuner/ListExportedSymbols.cs

```diff
@@ -125,8 +125,8 @@ void ProcessMethod (MethodDefinition method)
 				}
 			}
 
-			if (MarkStep.IsPropertyMethod (method)) {
-				var property = MarkStep.GetProperty (method);
+			if (method.IsPropertyMethod ()) {
```

Comment:
Curious - why were these code changes necessary?

Reply:
we use external source from monolinker repo - which is not quite API stable

Reply:
Changes in the API of the linker.

Reply:
👍